FeatureAreas.props

```diff
@@ -39,6 +39,7 @@
     <FeatureSliderEnabled>false</FeatureSliderEnabled>
     <FeatureSplitButtonEnabled>productOnly</FeatureSplitButtonEnabled>
     <FeatureSwipeControlEnabled>false</FeatureSwipeControlEnabled>
+    <FeatureTabViewEnabled>false</FeatureTabViewEnabled>
```

Comment:
I saw it at L12 😀

https://github.com/microsoft/microsoft-ui-xaml/blob/c7370860191925239e88b0eef3542de2eef618ae/FeatureAreas.props#L12

Reply:
oh no....

I'm really not sure how i missed that. I think changing the order to be alphabetical and match the official build declaration still makes sense though